### PR TITLE
Add sklearn-wrap

### PR DESCRIPTION
**Pull request by gtauzin**
## Summary

This PR adds a conda recipe for [sklearn-wrap](https://github.com/stateful-y/sklearn-wrap), a package for wrapping Python classes into Scikit-Learn estimators.

Checklist
 - [x]  Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
 - [x]  License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
 - [x]  Source is from official source.
 - [x]  Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
 - [x]  If static libraries are linked in, the license of the static library is packaged.
 - [x]  Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
 - [x]  Build number is 0.
 - [x]  A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
 - [x]  GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
 - [x]  When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by gtauzin** [comment]
I am willing to be listed as a maintainer.

**Comment by gtauzin** [comment]
It seems the win64 CI failure is unrelated to my recipe.

@conda-forge/help-python, ready for review!

File: recipes/sklearn-wrap/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
+{% set version = "0.1.0a4" %}
```
Comment (ocefpaf) [inline recipes/sklearn-wrap/meta.yaml:1]: This one falls under https://github.com/conda-forge/cfep/blob/main/cfep-05.md

Comment (gtauzin) [reply recipes/sklearn-wrap/meta.yaml:1]: @ocefpaf Hi, thanks for the review!

Regarding CFEP-05: sklearn-wrap has never had a stable (non-prerelease) release and 0.1.0a4 is the current intended published version, not a pre-release of an otherwise stable package. This falls under the CFEP-05 Exceptions clause:

"Certain packages (e.g. black) follow a release cycle in which they have never had a non-beta/alpha release. In these cases the conda packages do not need to be published to a prerelease label."

Once sklearn-wrap makes its first stable release, future pre-releases will go through the _dev/_rc label mechanism as required by CFEP-05.

Could you confirm whether this exception applies here?

Comment (ocefpaf) [reply recipes/sklearn-wrap/meta.yaml:1]: Black was published on PyPI as "stable" and, IMO, it was a terrible exception that we had to copy here to mimic the same behavior. With that said, this package is published on PyPI as pre-release, so publishing it here in a dev channel is necessary to keep some consistency between PyPI and conda-forge.

Screenshot From 2026-02-23 11-03-11
